### ISSUE TITLE
Error in Introduction Code: "Request is missing required query parameter 'user'"

### ISSUE THREAD

**Issue by porterdarby** [akka/akka-http#2915]
In the [Web Auction example](https://github.com/akka/akka-http/blob/v10.1.11/docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala#L517-L587) to the [Akka HTTP Introduction](https://doc.akka.io/docs/akka-http/current/introduction.html), I am running the code provided, but am receiving the following message when supplying the example curl request:

```commandLine
$ curl -X PUT http://localhost:8080/auction?bid=22&user=MartinO
[1] 41214
Request is missing required query parameter 'user'
$ 
```

If I switch the place of the parameters, the `bid` is considered missing:

```commandLine
$ curl -X PUT http://localhost:8080/auction?user=MartinO&bid=22
[1] 41342
Request is missing required query parameter 'bid'
$
```

I have tried a number of things to modify the request and the code so that it will render appropriately, even re-writing the code line for line manually so as to not potentially include any hidden characters.

**Additional Details**:
* I am running the application inside of IntelliJ IDE 2019.3.2
* The application is running off the scala-sdk-2.12.10
* I am using Apache Maven 3.6.3 for dependency management.
  * Scala Version 2.12.10
  * Scala Compat 2.12
  * Akka HTTP Version 10.1.11 for Scala 2.12 (Including Spray JSON, Parsing, and Core)

**Comment by johanandren** [comment]
Isn't that because you are not putting quotes around the url which means bash will take the part after & as a separate command and run the first part in the background?

**Comment by jrudolph** [comment]
> Isn't that because you are not putting quotes around the url which means bash will take the part after & as a separate command and run the first part in the background?

:+1: but that's because how we show it in the docs, sorry about that!

**Comment by jrudolph** [comment]
Fixed in #2907. Thanks for the PR, @porterdarby.
